addons/l10n_pt/models/account_move.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models, _, fields, api
+from odoo.exceptions import UserError
+from odoo.tools import float_repr
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    invoice_no = fields.Char(string='InvoiceNo', compute='_compute_invoice_no')
+
+    @api.depends('move_type', 'sequence_prefix', 'sequence_number')
+    def _compute_invoice_no(self):
+        for move in self:
+            move.invoice_no = move.move_type + ' ' + move.sequence_prefix[:-1].replace('/', '.') + '/' + str(move.sequence_number)
```

Comment:
Don't be afraid to use f-strings

Reply:
Why the `[:-1]`?

Reply:
Because we want to avoid having multiple `/` for the invoice_no. 
But this has been simplified now by a single replace :)